tasks/main.yml

```diff
@@ -33,15 +42,12 @@
   always:
     - name: Print out response
       debug:
-        var: snapshot_cmd_raw
+        var: snapshot_cmd
         verbosity: 2
 
-    - name: Parse raw output
+    - name: Set result
```

Comment:
I don't think you need to do this - I believe the `register: snapshot_cmd` in the `snapshot` task creates the global variable `snapshot_cmd`

Reply:
ok - fixed.